openapi-v1.yaml

```diff
@@ -6266,6 +6277,47 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
+  /files/{namespace}/upload:
+    parameters:
+      - type: string
+        name: namespace
+        in: path
+        required: true
+        description: The namespace of the file
+      - name: X-TILEDB-CLOUD-ACCESS-CREDENTIALS-NAME
+        in: header
+        description: Optional registered access credentials to use for creation
+        type: string
+        required: false
+    post:
```

Comment:
According to the [docs](https://swagger.io/docs/specification/2-0/file-upload/)

We need to add multipart/form-data

```
consumes:
  - multipart/form-data```